cloud_governance/common/elasticsearch/elasticsearch_operations.py

```diff
@@ -34,8 +35,10 @@ def __init__(self, es_host: str = None, es_port: str = None, region: str = '', b
         self.__es_port = es_port if es_port else self.__environment_variables_dict.get('es_port')
         self.__region = region
         self.__timeout = int(self.__environment_variables_dict.get('ES_TIMEOUT')) if self.__environment_variables_dict.get('ES_TIMEOUT') else timeout
-        self.__es = Elasticsearch([{'host': self.__es_host, 'port': self.__es_port}], timeout=self.__timeout, max_retries=2)
-
+        try:
+            self.__es = Elasticsearch([{'host': self.__es_host, 'port': self.__es_port}], timeout=self.__timeout, max_retries=2)
+        except:
+            pass
```

Comment:
is it ok to put pass here, dont we want to raise an error ?

Reply:
Even though elastic search fails. We need to send data to s3.  That's the reason I wrote pass there